### PR TITLE
feature/homepage-welcome: Work in progress for Welcome block.

### PR DESCRIPTION
**Pull request by montmadero**
Estoy pushando lo siguiente:
* Hero.
* Welcome Block: En progreso.

### CODE REVIEW

File: inc/homepage/homepage-welcome.inc [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      </div>
+    </div>
+    
+    <div class="cols-wrapper">
+      <div class="container-fluid" >
+        
+        <div class="row">
+          
+          <div class="column col-xs-12 col-sm-4 ">
+              <p>This website is for those who want to look great and feel great. Anyone can be healthier, have more energy, sleep better, and feel better about themselves through bodybuilding. Bodybuilding doesn't have to be expensive or take a lot of time. Our focus here is on workouts the natural bodybuilder can do at home with inexpensive equipment.
```
Comment (jlsanchezfr) [inline inc/homepage/homepage-welcome.inc]: Please fix the indentation for this line.

File: less/components/homepage/00-hero.less [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+          padding-top: 25px;
+          padding-bottom: 25px;
+          padding-left: 55px;
+          padding-right: 55px;
+          margin-top:0px; //overwrite tablet margin
+        }
+      }
+    }
+  }
+}
```
Comment (jlsanchezfr) [inline less/components/homepage/00-hero.less]: Please add a newline at the end of the file.

File: less/components/homepage/01-welcome-block.less [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+          padding-bottom: 25px;
+          padding-left: 45px;
+          padding-right: 45px;
+          background-color: #FFFFFF;//CTA
+          margin-bottom: 70px;
+        }
+      }
+    } 
+  }
+}
```
Comment (jlsanchezfr) [inline less/components/homepage/01-welcome-block.less]: Please add a newline at the end of the file.

**Comment by montmadero** [comment]
Thank you, all changes are fixed

**Pull request merged**